### PR TITLE
media-libs/libjxl: add 0.8.0

### PR DESCRIPTION
**Pull request by novomesk**

### CODE REVIEW

File: media-libs/libjxl/libjxl-0.8.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+		)
+	fi
+
+	cmake_src_configure
+}
+
+multilib_src_install() {
+	cmake_src_install
+
+	find "${D}" -name '*.a' -delete || die
```
Comment (ceamac) [inline media-libs/libjxl/libjxl-0.8.0.ebuild:72]:
```suggestion
	find "${ED}" -name '*.a' -delete || die
```

File: media-libs/libjxl/libjxl-0.8.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="openexr"
+
+DEPEND="app-arch/brotli:=[${MULTILIB_USEDEP}]
+	>=dev-cpp/highway-1.0.0[${MULTILIB_USEDEP}]
+	media-libs/giflib:=[${MULTILIB_USEDEP}]
+	>=media-libs/lcms-2.13:=[${MULTILIB_USEDEP}]
```
Comment (ceamac) [inline media-libs/libjxl/libjxl-0.8.0.ebuild:20]:
```suggestion
	media-libs/lcms:2[${MULTILIB_USEDEP}]
```

File: media-libs/libjxl/libjxl-0.8.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	media-libs/libjpeg-turbo[${MULTILIB_USEDEP}]
```
Comment (ceamac) [inline media-libs/libjxl/libjxl-0.8.0.ebuild:21]:
```suggestion
	media-libs/libjpeg-turbo:=[${MULTILIB_USEDEP}]
```

File: media-libs/libjxl/libjxl-0.8.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+	media-libs/libpng:=[${MULTILIB_USEDEP}]
+	sys-libs/zlib[${MULTILIB_USEDEP}]
```
Comment (ceamac) [inline media-libs/libjxl/libjxl-0.8.0.ebuild:23]: `qa-vdb` reports `sys-libs/zlib` this may not be needed.  Please check

**Review by ceamac — CHANGES_REQUESTED** [review]

File: media-libs/libjxl/libjxl-0.8.0.ebuild (hunk 2 above)
Comment (novomesk) [reply media-libs/libjxl/libjxl-0.8.0.ebuild]: I changed it to `>=media-libs/lcms-2.13:2[${MULTILIB_USEDEP}]`
I believe the version check is important there, some people may still have older version installed.

File: media-libs/libjxl/libjxl-0.8.0.ebuild (hunk 1 above)
Comment (novomesk) [reply media-libs/libjxl/libjxl-0.8.0.ebuild]: I changed it according your suggestion.

File: media-libs/libjxl/libjxl-0.8.0.ebuild (hunk 3 above)
Comment (novomesk) [reply media-libs/libjxl/libjxl-0.8.0.ebuild]: I changed the line as you suggested.

File: media-libs/libjxl/libjxl-0.8.0.ebuild (hunk 4 above)
Comment (novomesk) [reply media-libs/libjxl/libjxl-0.8.0.ebuild]: zlib is de facto included via libpng, so I removed it.

**Review by ceamac — APPROVED** [review]

**Comment by ceamac** [comment]
Thank you.

**Pull request closed without merging**